### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by Schwarzenegger**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Schwarzenegger/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
\ No newline at end of file
+# ReviewNinja-Welcome
+
+**Thanks for letting us show you around!**
+
```
Comment (Schwarzenegger) [inline ReadMe.md:4]: !fix This line

File: ReadMe.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+What you’re seeing here is a GitHub pull request with the code diffs
+showing by default. Feel free to look around.
+
+Over on the right is a panel for line notes that you can connect to a
+line of code in the diff. You can also switch to the conversation
+to see comments on the whole pull request.
```
Comment (Schwarzenegger) [inline ReadMe.md:10]: !resolve This

Comment (Schwarzenegger) [inline ReadMe.md:10]: !fixed

File: ReadMe.md (hunk 1 above)
Comment (Schwarzenegger) [inline ReadMe.md:4]: !fixed

**Pull request merged**